frontend/src/components/LandingPage.vue

```diff
@@ -211,7 +211,7 @@
           </template>
         </SmallCard>
       </v-col>
-      <v-col cols="12" :lg="isCCOFStatusNew ? 2 : 3">
+      <v-col cols="12" :lg="isCCOFStatusNew ? 4 : 4">
```

Comment:
I'd say this isn't necessary since the result is always 4.

Reply:
done.